Automattic/wp-calypso
# Login: Move login notices from global to the component

This pull request fixes part of #15101 by moving global notices when logging in to be at the top of the login form:
  
Before:
<img width="755" alt="screen shot 2017-06-15 at 14 50 31" src="https://user-images.githubusercontent.com/275961/27184418-0976f8fe-51da-11e7-846d-2974b41a51be.png">

After:
<img width="500" alt="screen shot 2017-06-15 at 14 47 07" src="https://user-images.githubusercontent.com/275961/27184436-121391e8-51da-11e7-95ea-0d7976a85106.png">

#### Testing instructions
  
1. Run `git checkout update/login-notices` and start your server
2. Open the [`Login` page](http://calypso.localhost:3000/log-in)
3. Check that you can still log in to a social account
4. Check that you see an error as above when unable to log in to a social account
  
#### Reviews
